packages/components/ShareModal/ShareModal.js

```diff
@@ -156,7 +158,9 @@ export const ShareModal = ({
   const shouldShowAddedMembers = members.length > 0;
   const shouldShowTeamsSection = false;
   const shouldShowSharedMembers = sharedMembers.length > 0;
-
+  const visibleEntitiesCount = items.length + members.length;
```

Comment:
what will happen if items are hidden? and when it will be toggled? how will height work?

Reply:
If items are hidden then the modal doesn't have simple wrapper rather than scrollbar component. Scrollvar appears only if modal height (with items) is enough to prevent dropdown shrink